python/ccxt/async_support/base/ws/aiohttp_client.py

```diff
@@ -71,6 +73,8 @@ def create_connection(self, session):
         # otherwise aiohttp's websockets client won't trigger WSMsgType.PONG
         # call aenter here to simulate async with otherwise we get the error "await not called with future"
         # if connecting to a non-existent endpoint
+        if (self.proxy):
```

Comment:
`proxy` are being set from outside explicitly (whenever needed), at i.e. 389th line of `async/exchange.py`.

Reply:
@ttodua shouldn't we return inside the if? otherwise we're calling `ws_connect` twice

Reply:
> @ttodua shouldn't we return inside the if? otherwise we're calling `ws_connect` twice

uh oh... right, i missed